modules/standard/IO.chpl

```diff
@@ -3319,50 +3318,94 @@ inline proc fileWriter.unlock() {
 }
 
 /*
-   Return the current offset of a fileReader.
+  A compile-time parameter to control the behavior of :proc:`fileReader.offset`
+  and :proc:`fileWriter.offset` when :param:`fileReader.locking` or
+  :param:`fileWriter.locking` is true.
 
-   .. warning::
+  When 'false', the deprecated behavior is used (i.e., calling
+  ``lock``/``unlock`` before getting the offset)
```

Comment:
Wording suggestion: `(i.e., the method acquires a lock internally before getting the offset)`